src/aura/BGE_EntryForm/BGE_EntryForm.cmp

```diff
@@ -68,26 +68,24 @@
 
         <lightning:layout multipleRows="true">
             <!--BEGIN SCOPED MATCH NOTIFICATION-->
-            <aura:if isTrue="{!or(not(empty(v.openOpportunities)), not(empty(v.unpaidPayments)))}">
-                <lightning:layoutItem size="12" smallDeviceSize="12" class="slds-p-horizontal_small">
-                    <div class="slds-scoped-notification slds-media slds-media_center slds-scoped-notification_light slds-m-bottom_small"
-                         role="alert">
-                        <div class="slds-media__figure slds-text-align_center">
-                            <lightning:icon iconName="utility:info"
-                                            alternativeText="{!$Label.c.PageMessagesInfo}"
-                                            size="x-small"
-                                            aria-hidden="true"/>
-                        </div>
-                        <div class="slds-media__body">
-                            <p>{!$Label.c.bgeMatchingAlert}
-                                <a class="slds-m-left_x-small" onclick="{!c.openMatchModal}" id="selectOpenDonation">
-                                    {!$Label.c.bgeMatchingSelect}
-                                </a>
-                            </p>
-                        </div>
+            <lightning:layoutItem size="12" smallDeviceSize="12" class="{!'slds-p-horizontal_small'+if(or(not(empty(v.openOpportunities)), not(empty(v.unpaidPayments))),'',' slds-hide')}">
```

Comment:
Is there a known issue about this somewhere. Would love to know why hide works but aura:if doesn't ...

Reply:
css leaves the markup in the dom but controls the visibility to the page, whereas aura:if actually prevents the markup from being included in the page until the condition is true. https://salesforce.stackexchange.com/questions/240348/auraif-vs-styling-to-show-hide-content

Reply:
I get that. Is this a problem everywhere we use aura:ifs, or is there something about this case that's different?

Reply:
I believe the combination of dynamically revealing that feature and trying to set focus on an element inside of it is relatively rare, and this particular issue is constrained to that combination. There's nothing inherently wrong with using aura:if outside of this particular set of circumstances.

Reply:
Cool. Thanks for clarifying! Given that, my only suggestion is maybe moving the hide onto a div outside the original component so it acts like an aura:if? That would make the intent of the styling clearer.

```
<div class="{!'if(or(not(empty(v.openOpportunities)), not(empty(v.unpaidPayments))),'',' slds-hide')}">
     <lightning:layoutItem size="12" smallDeviceSize="12" class="slds-p-horizontal_small">
```

Does that work?